### DISCUSSION TITLE
Why One to Many Relation is removed on v0.9.83

### DISCUSSION THREAD

**Discussion by masykur** [Budibase/budibase#2163] [General]
Hi,
I have upgraded budibase from v0.9.79 to v0.9.83. There is "one to many" relation feature was removed.
https://github.com/Budibase/budibase/compare/v0.9.79...v0.9.83#diff-e940c2181065c6e3b77d5dfad31e0a0bffd0df22f2ed6b4a88998d3cb0db8c47L175
The dropdown list become multi select, how to make it single selection?
Thank you

Masykur

**Comment by mjashanks** [comment]
Thanks for reporting! This is a bug, we will look into it asap. I opened an issue: https://github.com/Budibase/budibase/issues/2164
